Library/Homebrew/cask/migrator.rb

```diff
@@ -38,7 +39,9 @@ def migrate(dry_run: false)
       old_caskroom_path = old_cask.caskroom_path
       new_caskroom_path = new_cask.caskroom_path
 
-      old_installed_caskfile = old_cask.installed_caskfile.relative_path_from(old_caskroom_path)
+      return if (old_caskfile = old_cask.installed_caskfile).nil?
```

Comment:
```suggestion
      old_caskfile = old_cask.installed_caskfile
      return if old_caskfile.nil?
```

Don't feel strongly but this format makes more sense to me when the variable's reused below so the definition is not "hidden".